Class-6-(Inheritance).md

```diff
@@ -0,0 +1,19 @@
+# Class 6: Inheritance
+
+## In the prep work this week, the students learned
+1. What inheritance is and how programmers can use it.
+1. What abstract classes and methods are.
+1. How to formally diagram classes and the relationships between classes.
+
+## Announcements
+1. 
```

Comment:
i'd just leave this section out or say like, 'check with the course manager' otherwise it can look like a mistake that its here

Reply:
Got it! Thanks!